parsons/action_network/action_network.py

```diff
@@ -1858,26 +1858,42 @@ def create_tagging(self, tag_id, payload):
                         "osdi:person" : { "href" : "https://actionnetwork.org/api/v2/people/id" }
                     }
                 }
+            background_processing: bool
+                If set `true`, utilize ActionNetwork's "background processing". This will return
+                an immediate success, with an empty JSON body, and send your request to the
+                background queue for eventual processing.
+                https://actionnetwork.org/docs/v2/#background-processing
         `Returns:`
             A JSON response after creating the tagging
         `Documentation Reference`:
             https://actionnetwork.org/docs/v2/taggings
         """
-        return self.api.post_request(f"tags/{tag_id}/taggings", data=json.dumps(payload))
+        url = f"tags/{tag_id}/taggings"
+        if background_processing:
+            url = f"{url}?background_processing={background_processing}"
```

Comment:
is the value passed to the param case insensitive?

Reply:
It looks like we have methods that behave both ways, some that pass "background_processing=true" and some that pass "background_processing={background_processing}".  Since update_person and upsert_person use "background_processing=true" I've updated the code here to do the same.